Packs/CommonPlaybooks/Playbooks/playbook-Cloud_Response_-_Generic.yml

```diff
@@ -422,36 +410,40 @@ tasks:
       '#none#':
       - "20"
     scriptarguments:
+      accessKeyName:
+        complex:
+          root: inputs.accessKeyName
       accessKeyRemediationType:
-        simple: Disable
+        complex:
+          root: inputs.GCP-accessKeyRemediationType
       autoAccessKeyRemediation:
-        simple: "False"
+        complex:
+          root: inputs.autoAccessKeyRemediation
       autoBlockIndicators:
-        simple: "False"
+        complex:
+          root: inputs.autoBlockIndicators
       autoResourceRemediation:
-        simple: "False"
+        complex:
+          root: inputs.autoResourceRemediation
       autoUserRemediation:
-        simple: "False"
+        complex:
+          root: inputs.GCP-accessKeyRemediationType
       resourceName:
         complex:
           root: alertJson._all_events
```

Comment:
that one stayed with `alertJson._all_events `context path

Reply:
fixed